### DISCUSSION TITLE
Configuration for UpperCamelCase

### DISCUSSION THREAD

**Discussion by briellelaw** [strawberry-graphql/strawberry#3153] [Q&A]
Is there a configuration setting to force UpperCamelCase instead of lowerCamelCase?

I see there is a closed issue that introduced a auto_capitalize bool (https://github.com/strawberry-graphql/strawberry/pull/548), but this seems outdated with the introduction of StrawberryConfig (https://strawberry.rocks/docs/types/schema-configurations)

**Comment by patrick91** [comment]
hi there! 

you can still use auto_camel_case=False, like here: https://play.strawberry.rocks/?gist=c8b6246d584eb2c20f75faa6e651ab19, but that means that you need to write the fields exactly like you want them in python 

the other option is to use the name converter, we haven't document it yet, mostly because I'm not 100% sure of the API (but I don't think we'll make any breaking changes to it), like here: https://play.strawberry.rocks/?gist=61ec081ffeb5272902e3eecbf96dc202
